group_vars/model_zyxel_nwa55axe.yml

```diff
@@ -0,0 +1,17 @@
+---
+target: ramips/mt7621
+
+dsa_ports:
+  - lan
```

Comment:
Since this a single port device you dont have to configure the switch. IMO you should better use
```
int_port: lan
```

Reply:
will do